app/models.py

```diff
@@ -105,6 +111,15 @@ def validate_phone_number(form, field):
             logger.debug(" There is a number parse exception in the phonenumber field")
             raise ValidationError('This should be a valid UK number e.g. 01632 496 0018. ')
 
+    def validate_password(form, field):
+        if len(field.data) > Config.PASSWORD_MAX_LENGTH:
```

Comment:
Thanks for the feedback. This is similar to what's in place in the **validate_phone_number** function as well. I think this in place as a 'catch-all' just in case someone makes a request to the service via cURL, as the field validation would be bypassed. I do need to implement a check for length < PASSWORD_MAX_LENGTH now though as I seemed to have forgotten to do this here!